.gitignore

```diff
@@ -106,4 +106,6 @@ venv.bak/
 #vscode
 .vscode/
 
-report_unittest.xml
\ No newline at end of file
+report_unittest.xml
+
+e-commerce/
```

Comment:
Why this in gitignore?

Reply:
i created my venv in E-commerce folder so Gurkirat told me to add it in this